tests/frontend/gpu/floydwarshall.cpp

```diff
@@ -0,0 +1,330 @@
+// NOLINTBEGIN
+
+// clang-format off
+// RUN: rm -rf .proteus
+// RUN: ./floydwarshall.%ext 1024 100 16 | %FILECHECK %s --check-prefixes=CHECK,CHECK-FIRST
```

Comment:
Add `PROTEUS_TRACE_OUTPUT=1` and check that we get the specialization printout

Reply:
It looks like the frontend doesn't go through the specialization. It goes `Dispatch.compile` -> (dispatcher) `Jit.compileOnly`.